app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -243,6 +169,129 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
         }
     }
 
+    private fun setScreenSize(isTablet: Boolean) {
+        if (isTablet == this.isTablet) return
+
+        if (isTablet) {
+            addTabletActions(requireContext())
+        } else {
+            removeTabletActions()
+        }
+
+        this.isTablet = isTablet
+    }
+
+    @Suppress("LongMethod")
+    private fun addTabletActions(context: Context) {
+        val enableTint = ThemeManager.resolveAttribute(R.attr.textPrimary, context)
+        val disableTint = ThemeManager.resolveAttribute(R.attr.textDisabled, context)
+        backAction?.let {
+            browserToolbarView.view.addNavigationAction(it)
+        } ?: kotlin.run {
```

Comment:
```suggestion
        } ?: run {
```
We can shorten this to `run` for all your actions here. I am mostly asking for this change because we use `run` everywhere else.